.taskcluster.yml

```diff
@@ -0,0 +1,42 @@
+version: 1
+policy:
+  pullRequests: collaborators
+tasks:
+  $let:
+    head_rev:
+      $if: tasks_for == "github-pull-request"
+      then: ${event.pull_request.head.sha}
+      else: ${event.after}
+    repository:
+      $if: tasks_for == "github-pull-request"
+      then: ${event.pull_request.head.repo.html_url}
+      else: ${event.repository.html_url}
+  in:
+    $match:
+      (tasks_for == "github-pull-request" && event["action"] in ["opened","reopened","synchronize"]):
+        taskId:
+          $eval: as_slugid("test")
+        deadline:
+          $fromNow: 1 day
+        taskQueueId: proj-misc/tutorial
```

Comment:
I've never set this one myself, do you know what it is exactly?

Reply:
@marco-c I'm not totally sure, honestly. I know that the `proj-misc/tutorial` part refers to a [worker pool](https://community-tc.services.mozilla.com/provisioners/proj-misc/worker-types/tutorial?sortBy=Last%20Active&sortDirection=desc) of 4 workers in `us-east`, but I don't know beyond that. That section was auto-generated by the [Quickstart page](https://community-tc.services.mozilla.com/quickstart).

Reply:
Maybe it is a union of `provisionerId` and `workerType` that we are using in other projects, e.g. for https://github.com/mozilla/libmozdata/blob/33a55d7876cc44a021b34b920b0637184acbeb37/.taskcluster.yml#L46-L47 it would be `proj-relman/ci`.

Reply:
Let's try to change it to `proj-relman/ci` then, to avoid using the tutorial one